### PR TITLE
Update Foxy.rst

### PR DESCRIPTION
**Pull request by drhieu**
Add AMCL Interface Changes #1945 on Navigation

### CODE REVIEW

File: migration/Foxy.rst [hunk 1]
```diff
@@ -23,3 +23,8 @@ BackUp Recovery Interface Changes
 ``speed`` in a backup recovery goal should be positive indicating the speed with which to drive backward.
 ``target.x`` in a backup recovery goal should be positive indicating the distance to drive backward.
 In both cases negative values are silently inverted.
+
+AMCL Interface Changes
+*********************************
+
+The parameters name for ``base_frame_id``, ``global_frame_id``, and ``odom_frame_id`` are now ``base_frame``, ``global_frame``, and ``odom_frame`` so it would be consistent with other packages.
```
Comment (SteveMacenski) [inline migration/Foxy.rst:30]: You need to update the parameter guide https://navigation.ros.org/configuration/index.html

**Comment by SteveMacenski** [comment]
@drhieu please make these updates so we can merge this.

File: migration/Foxy.rst (hunk 1 above)
Comment (drhieu) [reply migration/Foxy.rst:30]: Updated through this PR

https://github.com/ros-planning/navigation.ros.org/pull/86

Comment (SteveMacenski) [reply migration/Foxy.rst:30]: No, please add it to this PR.

**Comment by SteveMacenski** [comment]
Closing from inactivity.

**Pull request closed without merging**